65black/StackOverflow-lite
# #159964115 Add JSDoc and Swagger Documentation

#### What does this PR do?
This PR adds documentation to the server side using JSDoc. 
The link to the Swagger Documentation is also included. There are instructions on how to run the app on the local machine added to the README as well.

#### How should this be manually tested?
 - Open Terminal and navigate to the root folder of the repo.
 - Type in the command and run it: `npm run documentation`.
 - Notice the new folder called docs. 

#### What are the relevant pivotal tracker stories?
#159964115
